ui/src/App.module.css

```diff
@@ -107,13 +101,180 @@
     width: 100%;
   }
 
-  :global(.fc-toolbar) {
-    display: block !important;
-    text-align: center;
+  /* DIV FC TOOLBAR CHUNK */
+  :global(
+      #root
+        > div
```

Comment:
Would this break if there's a new div added to the DOM?

Reply:
Would be good to simplify this if possible

Reply:
Nice, just simplified this to grab only what's necessary.